packages/create-neutrino-project/commands/init/generator/writing.js

```diff
@@ -0,0 +1,158 @@
+const deepmerge = require('deepmerge');
+const mkdirp = require('mkdirp');
+const path = require('path');
+const changeCase = require('change-case');
+const utils = require('./utils');
+
+const VERSIONS = utils.VERSIONS;
+const PROJECTS = utils.PROJECTS;
+const LINTERS = utils.LINTERS;
+
+function makeRcFile(data) {
+  let config = { use: [] };
+
+  if (data.projectType === PROJECTS.REACT) {
+    config.use.push('neutrino-preset-react');
+  } else {
+    config.use.push(`neutrino-preset-${data.projectType}`);
+  }
+
+  if (data.testRunner) {
+    config.use.push(`neutrino-preset-${data.testRunner}`);
+  }
+
+  switch (data.linter) {
+    case LINTERS.AIRBNB:
+      config.use.push(`neutrino-preset-${data.linter}${data.projectType === PROJECTS.WEB ? '-base' : ''}`);
+      break;
+    case LINTERS.STANDARDJS:
+      config.use.push(`neutrino-middleware-${LINTERS.STANDARDJS}`);
+      break;
+    default:
+      config.use.push(`neutrino-preset-${data.linter}`);
+  }
+
+  return `module.exports = ${JSON.stringify(config, null, 2)};\n`;
```

Comment:
Instead of `JSON.stringify`, let's try the `javascript-stringify` package that Neutrino also uses.